src/Client/SearchClient.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Auditing\Client;
+
+use LoyaltyCorp\Search\Interfaces\ClientInterface;
+
+abstract class SearchClient implements ClientInterface
```

Comment:
What is the purpose of this abstract class? Does it need documentation?

Reply:
Removed. Not required anymore.